### ISSUE TITLE
Where does the package advtools comes from.

### ISSUE THREAD

**Issue by VVebSharE** [jeromerony/augmented_lagrangian_adversarial_attacks#2]
->models/imagenet.py: line 5; 
 from advtools.utils import normalize_model, requires_grad_

What is advtools here

**Comment by jeromerony** [comment]
Hi,
`advtools` does not correspond to any published package. It was the initial name of the adversarial library package before it was released. It can be replaced by `adv_lib`.

**Comment by VVebSharE** [comment]
Thanks for the quick resolution
